packages/@cdktf/provider-generator/lib/get/generator/models/attribute-model.ts

```diff
@@ -47,6 +47,8 @@ export function escapeAttributeName(name: string) {
   if (name === "system") return "systemAttribute";
   // `tfResourceType` is already used by resources to distinguish between different resource types
   if (name === "tfResourceType") return `${name}Attribute`;
+  // `importFrom` has potential for common name collision with providers
+  if (name === "importFrom") return `${name}Provider`;
```

Comment:
I think, I'd name it to `importFromAttribute` (same as for `self`, `build`, `node`, `system`, ... 😉)
```suggestion
  if (name === "importFrom") return `${name}Attribute`;
```

Reply:
👍